tests/Integration/PrestaShopBundle/Controller/TestEntityDTO.php

```diff
@@ -69,12 +69,24 @@ public function getId(): ?int
      *
      * @throws VariableNotFoundException
      */
-    public function __get(string $variableName)
+    public function getVariable(string $variableName)
     {
         if (!isset($this->variables[$variableName])) {
             throw new VariableNotFoundException(sprintf('Variable %s not found in entity', $variableName));
         }
 
         return $this->variables[$variableName];
     }
+
```

Comment:
didn't notice this TestEntityDTO class before. Naming is so strange. It should either be Entity or a DTO, but not both (or is it like a DTO for Testing the entity? :smile: Anyway just TestEntity would be enough I guess..)

Reply:
Actually, that's what you said it's a generic DTO used to test any entity 😅 So the naming isn't that bad, I don't mind it but if you want me to change it I don't mind either But TestEntity makes it sound like it's a specific Entity that would have been implemented as a specific use case for test, whereas this is a generic structure that can be used to test any entity